lib/util.js

```diff
@@ -49,18 +49,15 @@ let util = {
         let size = parseInt(stdout.toString().split(/\s+/)[0]);
         if (Number.isNaN(size)) size = 0;
 
-        if (size && error)
-        {
+        if (size && error) {
```

Comment:
Move the opening brace to the next line to maintain consistency with the rest of the project i.e.
```js
if (size && error)
{
  // ...
}
else
{
  // ...
}
```
This needs to be fixed in several places in this commit.

Keep in mind that in this project, array functions do NOT follow this style. So arrow functions should still look like:
```js
.then(value => {
  // ...
});
```